backend/tests/Feature/StyleCriteriaTest.php

```diff
@@ -1,11 +1,11 @@
 <?php
-declare(strict_types=1);
+    declare(strict_types=1);
```

Comment:
```suggestion
declare(strict_types=1);
```
I've only ever seen these unindented in our codebase.